src/codegen/sys/lib_.rs

```diff
@@ -106,12 +107,12 @@ fn generate_bitfields<W: Write>(w: &mut W, items: &[&Bitfield])
     try!(writeln!(w, ""));
     for item in items {
         try!(writeln!(w, "bitflags! {{\n{}#[repr(C)]\n{0}flags {}: c_uint {{",
-                      tabs(1), item.c_type));
+                      TAB, item.c_type));
         for member in &item.members {
             let val: i64 = member.value.parse().unwrap();
-            try!(writeln!(w, "{}const {} = {},", tabs(2), member.c_identifier, val as u32));
+            try!(writeln!(w, "{}{0}const {} = {},", TAB, member.c_identifier, val as u32));
```

Comment:
We could use named parameters for tabs to make this less confusing:
```rust
try!(writeln!(w, "{tab}{tab}const {} = {},", member.c_identifier, val as u32, tab = TAB));
```
Or perhaps for all arguments.